src/server/pfs/server/master.go

```diff
@@ -35,18 +35,17 @@ const (
 
 type Master struct {
 	env Env
-
-	driver *driver
+	*apiServer
```

Comment:
This doesn't seem like a good idea.  There's probably only a few methods that overlap, we shouldn't need to have a whole apiServer to share methods.

Reply:
I agree, I think a good follow up would be to define a struct that just extracts what is shared. This would be like a rescoped driver_v2 or something.